frontend/src/app/core/services/user.service.ts

```diff
@@ -19,4 +19,10 @@ export class UserService {
   registerUser(userForRegistration: UserForRegistrationDTO): Observable<void> {
     return this.httpClient.post<void>(`${this.registrationPageWebAPIUrl}/register`, userForRegistration);
   }
+
+  getUser(id: string): Observable<UserDTO> {
+    let params = new HttpParams().set('id', id);
+    
+    return this.httpClient.get<UserDTO>(`${this.registrationPageWebAPIUrl}/user`, { params: params });
```

Comment:
shouldn't we include authentication token?

Reply:
we don't have authentication implemented yet